README.md

```diff
@@ -14,23 +131,28 @@ pnpm dev
 bun dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the webpage running locally.
+
+
 
-You can start editing the page by modifying `app/page.js`. The page auto-updates as you edit the file.
+**👉 [Read our detailed Contributing Guide](.github/CONTRIBUTING.md)** for complete setup instructions and development workflow.
 
-This project uses [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to automatically optimize and load [Geist](https://vercel.com/font), a new font family for Vercel.
+## 🔐 Security and Privacy
 
-## Learn More
+- **No Credential Storage**: The website does not store Discord or PESU passwords
+- **Secure Database**: All data is stored securely in MongoDB with proper access controls
+- **Role-based Access**: Commands are restricted based on user permissions and server roles
 
-To learn more about Next.js, take a look at the following resources:
+## 📊 Project Status
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+- **Active Development**: The website is actively maintained and updated
+- **Community Driven**: Features are developed based on community needs
+- **Production Ready**: Currently deployed and serving the PESU Discord community
```

Comment:
remove this. doesn't serve any important purpose. instead add one sentence in the top of the README that says something along the lines of "this project is being actively maintained and updated"